Apps/Src/main.c

```diff
@@ -56,26 +54,18 @@ int main(void) {
     assertOSError(err);
 
     while(1);
-
-    return 0;
 }
 
 void Task_Init(void *p_arg){
     OS_ERR err;
 
     // Start systick    
     OS_CPU_SysTickInit(SystemCoreClock / (CPU_INT32U) OSCfg_TickRate_Hz);
-
-    OSTimeDlyHMSM(0,0,5,0,OS_OPT_TIME_HMSM_STRICT,&err);
-
-    assertOSError(err);
     
     // Initialize drivers
     Pedals_Init();
-    OSTimeDlyHMSM(0,0,5,0,OS_OPT_TIME_HMSM_STRICT,&err);
-    OSTimeDlyHMSM(0,0,10,0,OS_OPT_TIME_HMSM_STRICT,&err);
     BSP_UART_Init(UART_2);
-    CANbus_Init(CARCAN, (CANId_t*)carCANFilterList, NUM_CARCAN_FILTERS);
+    CANbus_Init(CARCAN, carCANFilterList, NUM_CARCAN_FILTERS);
     CANbus_Init(MOTORCAN, NULL, NUM_MOTORCAN_FILTERS);
```

Comment:
Are we using `motorCANFilterList` here?

Reply:
We disabled motorCANFilterList during testing and re-enabling it now would probably mean we'd have to re-test. I'll make an issue ticket for re-testing this functionality on main when we get back.

Reply:
#397